scripts/spot-grid.py

```diff
@@ -36,14 +37,16 @@ class VideoDisplayerGrid(VideoDisplayer):
     It should be pretty much platform independent.
     """
 
-    def __init__(self, title="Live image", size=(640, 480), gridsize=None):
+    def __init__(self, title="Live image", size=(640, 480), gridsize=None,
+                 pxsize=PIXEL_SIZE_SAMPLE_PLANE / DEFAULT_MAGNIFICATION):
```

Comment:
I'm not a fan of having calculations as default arguments, it is a bit hard to read. Why not create a variable PIXEL_SIZE at the top?

Reply:
Now that I've read the rest of the code, you could also have pxsize as a required argument, and not give it a default value.

Reply:
You mean the 3.45 µm (pixel size sample plane) as an additional argument? Or have the pixelsize argument instead of the magnification (so the user would directly enter 3.45e-6/40 for the pixelsize)?

Reply:
I think she means: "leave pxsize, but drop the default value". At least, that's what I'd suggest to do.

Reply:
Indeed I meant "leave pxsize, but drop the default value". The signature would become:
```
    def __init__(self, pxsize, title="Live image", size=(640, 480), gridsize=None):
```

Reply:
@pieleric Just discussed with Thera, my issue with that was that if we do it like this, the hardware argument pxsize would come before the gui arguments (title, size) and that might be confusing? Maybe it's not such a big deal. What do you think? Another option is to do the division in another constant, so it looks a little cleaner.